src/NServiceBus.Gateway.Tests/ApprovalFiles/APIApprovals.Approve.netstandard.approved.txt

```diff
@@ -38,11 +38,15 @@ namespace NServiceBus.Gateway
     {
         System.Threading.Tasks.Task Send(string remoteAddress, System.Collections.Generic.IDictionary<string, string> headers, System.IO.Stream data);
     }
+    public interface IDuplicationCheckSession : System.IDisposable
+    {
+        bool IsDuplicate { get; }
+        System.Threading.Tasks.Task MarkAsDispatched();
+    }
     public interface IGatewayDeduplicationStorage
     {
         bool SupportsDistributedTransactions { get; }
```

Comment:
Rename to `SupportsTransactionScope`?

Reply:
I think that wouldn't capture the intention  correctly. While a transaction scope is used, we only use a transaction scope when a distributed transaction should be used. What if a storage could enlist with a transaction scope but not with distributed transactions?

Reply:
I'm not sure if that would change the outcome of the behavior. I don't know the ins/ours ot transaction scope implementation behavior principles.